exonum/src/runtime/rust/mod.rs

```diff
@@ -14,11 +14,11 @@
 
 //! The current runtime is for running native services written in Rust.
 //!
-//! A set of artifacts available to deploy in the Rust runtime is static. The set is defined at the time
-//! of compilation. Once created, the set can be changed only by the node binary recompilation.
+//! In the Rust runtime a set of service artifacts that you want to deploy is static. The set is defined at the time
```

Comment:
Nit: IMO, "you may want to deploy" would look more accurate. During blockchain initialization, a user may not instantiate services from all available artifacts; the corresponding services may be instantiated later.